### DISCUSSION TITLE
No valid solidity source files found.

### DISCUSSION THREAD

**Discussion by AshmeetSingh987** [Cyfrin/security-and-auditing-full-course-s23#123] [Q&A] [answered]
After installing Solidity Metrix  , on compiling am getting error of  No valid solidity source files found. on Src folder , if anyone has any idea to resolve please do let me know Thanks

**Comment by tobezhanabi** [comment]
whats your folder, take a screenshot

**Reply by AshmeetSingh987** [reply]
![image](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/69580963/8ea83240-5c49-417d-a471-436433d80209)

**Comment by alymurtazamemon** [comment]
@AshmeetSingh987 Make sure you have a `src` folder (check the capitalization of letters) and inside it, you should have `.sol` files.

**Reply by AshmeetSingh987** [reply]
yes have checked its .sol only

**Reply by alymurtazamemon** [reply, marked as answer]
@AshmeetSingh987 Please check this [link](https://github.com/Consensys/vscode-solidity-metrics/issues/19)

**Reply by AshmeetSingh987** [reply]
Thanks bro it worked
